portfolio/src/main/webapp/index.html

```diff
@@ -3,8 +3,8 @@
   <head>
     <meta charset="UTF-8">
     <title>more about B</title>
-    <link rel="stylesheet" href="/style.css">
-    <script src="/script.js"></script>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
```

Comment:
I still think you'll need the leading slashes here in order for the container to render the style correctly. Have you tried invoking `mvn package appengine:run` to verify this?

Reply:
theres a preview in the cloud shell that you can run quickly to see what your changes look like, and when i had the leading slashes, i couldnt see them, but if i removed them i could quickly view the styling changes. 

i forgot to add them back before pushing the code, but yes the leading slashes still render the page correctly. :)

Reply:
Huh - when I tried this yesterday with your repo it wasn't rendering styles when running in the container - but indeed I am incorrect :). Just tried it for my own sanity and it seems to work either way. Feel free to leave it or revert.